app/models/opentofu_worker.rb

```diff
@@ -49,6 +49,13 @@ def unit_environment_variables
     }
   end
 
+  def configure_service_worker_deployment(definition)
+    super
+
+    definition[:spec][:template][:spec][:containers].first[:volumeMounts] << {:name => "cert-path", :mountPath => "/root"}
+    definition[:spec][:template][:spec][:volumes] << {:name => "cert-path", :secret => {:secretName => "application-certificate-secret", :items => [{:key => "httpd_crt", :path => "server.crt"}, {:key => "httpd_key", :path => "server.key"}], :defaultMode => 420}}
```

Comment:
`application-certificate-secret` should not be hard-coded.

Reply:
`{:key => "httpd_crt"` This should have its own certificate, not reuse the one for httpd.

Reply:
Updated IM installer code to have new key and cert in secret `application-certificate-secret`.
Using those secret key for creating new volume mounts.
ref: https://github.ibm.com/IBMPrivateCloud/ibm-infra-management-install-operator/pull/566/files

Reply:
There is no environment variable I can use for reading secret name as `application-certificate-secret`. 
May be I am missing existing so please guide. Secondly If I define the variable and assign value to it as `application-certificate-secret`  it is as good as hard coding.

Reply:
Using environment variable API_SSL_SECRET_NAME instead of actual secret name